addons/60fps/addon.json

```
@@ -1,10 +1,10 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Alt+click green flag to change project framerate",
+  "description": "Lets you run projects at higher framerates so they look smoother. Hold Alt and click the green flag to toggle between Scratch's default 30 FPS and a custom framerate set below.",
```

To abbreviate "frames per second" in the description:
```suggestion
  "description": "Lets you run projects at higher framerates so they look smoother. Hold Alt and click the green flag to toggle between Scratch's default 30 frames per second (FPS) and a custom framerate set below.",
```